v1/python-sdk/tutorials/automl-with-azureml/forecasting-backtest-many-models/auto-ml-forecasting-backtest-many-models.ipynb

```diff
@@ -492,25 +524,29 @@
     "\n",
-    "#### get_many_models_batch_inference_steps arguments\n",
+    "#### ``get_many_models_batch_inference_steps`` arguments\n",
     "| Property                           | Description|\n",
     "| :---------------                   | :------------------- |\n",
     "| **experiment**                     | The experiment used for inference run. |\n",
     "| **inference_data**                 | The data to use for inferencing. It should be the same schema as used for training.\n",
-    "| **compute_target**                 | The compute target that runs the inference pipeline.|\n",
+    "| **compute_target**                 | The compute target that runs the inference pipeline. |\n",
     "| **node_count**                     | The number of compute nodes to be used for running the user script. We recommend to start with the number of cores per node (varies by compute sku). |\n",
-    "| **process_count_per_node**         | The number of processes per node.\n",
-    "| **train_run_id**                   | \\[Optional\\] The run id of the hierarchy training, by default it is the latest successful training many model run in the experiment. |\n",
-    "| **train_experiment_name**          | \\[Optional\\] The train experiment that contains the train pipeline. This one is only needed when the train pipeline is not in the same experiement as the inference pipeline. |\n",
-    "| **process_count_per_node**         | \\[Optional\\] The number of processes per node, by default it's 4. |"
+    "| **process_count_per_node**         | \\[Optional] The number of processes per node, by default it's 2 (should be at most half of the number of cores of the compute cluster that will be used for the experiment).\n",
+    "| **inference_pipeline_parameters**  | \\[Optional] The ``ManyModelsInferenceParameters`` object defined above. |\n",
+    "| **append_row_file_name**           | \\[Optional] The name of the output file (optional, default value is 'parallel_run_step.txt'). Supports 'txt' and 'csv' file extension. A 'txt' file extension generates the output in 'txt' format with space as separator without column names. A 'csv' file extension generates the output in 'csv' format with comma as separator and with column names. |\n",
+    "| **train_run_id**                   | \\[Optional] The run id of the **training pipeline**, by default it is the latest successful training many model run in the experiment. |\n",
+    "| **train_experiment_name**          | \\[Optional] The train experiment that contains the train pipeline. This one is only needed when the train pipeline is not in the same experiement as the inference pipeline. |\n",
+    "| **run_invocation_timeout**         | \\[Optional] Maximum amount of time in seconds that the ``ParallelRunStep`` class is allowed. This is optional but provides customers with greater control on exit criteria. |\n",
```

Comment:
Should we add a note here that it must be greater than the experiment timeout (in seconds) in the AutoML Config?

Reply:
I don't think this is required in case of inferencing, ``run_invocation_timeout`` is independent of ``experiment_timeout_hours`` in case of inferencing.

@yuanzhuangyuanzhuang, please correct me if I am mistaken.

Reply:
We need to add this statement. This is from the current version: 
![image](https://user-images.githubusercontent.com/65432549/205723541-9b68015a-71c9-4481-915c-dce94d54e507.png)

Reply:
I see, but this is from training, ``run_invocation_timeout`` param in ``get_many_models_train_steps``. We already have this for training. Inference (``get_many_models_batch_inference_steps``) does not require this explanation as we don't have any ``experiment_timeout_hours`` while doing inferencing.